phy/cluster/default_settings.py

```diff
@@ -7,47 +7,48 @@
 # Spike detection
 # -----------------------------------------------------------------------------
 
-def spikedetekt_params(sample_rate):
-
-    assert sample_rate > 0
+traces = {
+    'raw_data_files': [],
+    'n_channels': None,
+    'dtype': None,
+    'sample_rate': None,
+}
 
-    return dict(
-        # Filter.
-        filter_low=500.,
-        filter_high=0.95 * .5 * sample_rate,
-        filter_butter_order=3,
 
-        # Data chunks.
-        chunk_size=int(1. * sample_rate),
-        chunk_overlap=int(.015 * sample_rate),
+spikedetekt = {
+    'filter_low': 500.,
+    'filter_high_factor': 0.95 * .5,  # will be multiplied by the sample rate
```

Comment:
Aha, now I understand. This is simply an anti-aliasing filter.

Could we instead rename this parameter to aa_nyquist_filter_factor to make its purpose more explicit? Confusingly it's a lowpass filter. 

In fact I would personally vote to have this as a fixed frequency (0.47*sample_rate) and just set one variable aa_filter = True/False - why would we really want to change this? @nsteinme